dash_bio/package-info.json

```diff
@@ -1 +1 @@
-{"name": "dash_bio", "version": "0.0.11", "author": "The Plotly Team"}
\ No newline at end of file
+{"name": "dash_bio", "version": "0.1.0rc2", "author": "The Plotly Team"}
```

Comment:
@shammamah I'm not sure I've seen version `0.1.0rc1` -- is there another PR I should be approving before this one?

Reply:
Nope! I was just testing stuff out.

Reply:
It was in this PR: d62dbcd1f77a41b23ca35f516801ad04ca341b57

Reply:
That's a commit, not a PR... even inspecting the commit history around May 16th (date of commit d62dbcd1f77a41b23ca35f516801ad04ca341b57), I can't figure out which PR it was submitted with, but anyhow.